Android/app/src/main/java/io/github/project_travel_mate/destinations/description/PlacesOnMapActivity.java

```diff
@@ -89,7 +89,8 @@
     BottomSheetBehavior sheetBehavior;
     @BindView(R.id.bottom_sheet)
     LinearLayout layoutBottomSheet;
-    private MapView mMap;
+    @BindView(R.id.map)
+    MapView mMap;
```

Comment:
this change was intended for all places. Only Private variables start with `m...`